tough/Cargo.toml

```diff
@@ -11,6 +11,7 @@ edition = "2018"
 [dependencies]
 async-recursion = "1"
 async-trait = "0.1"
+aws-lc-rs = { version = "1", features = ["prebuilt-nasm"] }
```

Comment:
Correct, this is only a feature for the Windows platform

https://github.com/aws/aws-lc-rs/blob/8d428026db8a8d4434d3db50e982ad65a98fdeb3/book/src/requirements/windows.md?plain=1#L11-L12

> Windows requirements
> [...]
> The NASM assembler is recommended on `x86-64` platforms. NASM is required for `x86` and for "fips" builds. 

Setting the feature pulls in the project's NASM objects rather than falling back to those locally available in the environment: https://github.com/aws/aws-lc-rs/tree/main/aws-lc-sys/builder/prebuilt-nasm https://docs.rs/aws-lc-rs/latest/aws_lc_rs/#prebuilt-nasm

Reply:
If you'd prefer we leave this up to consumers of `tough`, I could see about what's needed to add NASM to our CI (could split out the Windows CI and set `AWS_LC_SYS_PREBUILT_NASM=1`

Reply:
I believe on non-windows platforms the feature is a no-op.

> Enables the use of crate provided prebuilt NASM objects under certain conditions. This only affects builds for Windows x86-64 platforms. This feature is ignored if the "fips" feature is also enabled.